lib/railway_ipc/consumer/consumer.rb

```diff
@@ -56,8 +56,14 @@ def exchange_name
       queue.opts[:exchange]
     end
 
-    def work(payload)
-      message = RailwayIpc::IncomingMessage.new(payload)
+    # REFACTOR: Long term we should think about not leaking Sneakers
+    # methods as part of Railway's public API since clients can (and do)
+    # override them. -BN
+    def work_with_params(payload, _delivery_info, metadata)
```

Comment:
i'm assuming this is currently underscored because it's not currently being used..

Reply:
Correct. Rubocop convention is to either use a plain `_` or the variable name prefixed with a `_`.